### PR TITLE
WIP Payment history content misuv 3698

### PR DESCRIPTION
**Pull request by singapaul**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -104,6 +104,7 @@ home.updates.heading                                            = Next updates d
 home.updates.view                                               = View update deadlines
 home.paymentHistory.heading                                     = Payment history
 home.paymentHistory.view                                        = Payment history
+home.paymentHistoryRefund.heading                               = Payment and refund history
```
Comment (joesheridan) [inline conf/messages:107]: is this needed? I though Vinay had this covered?

File: it/testConstants/messages/PaymentHistoryMessages.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package testConstants.messages
+
+import helpers.servicemocks.AuthStub.{lang, messagesAPI}
+
+object PaymentHistoryMessages {
+
+  val paymentHistoryHeading: String = messagesAPI("Payment and refund history")
```
Comment (joesheridan) [inline it/testConstants/messages/PaymentHistoryMessages.scala:22]: I'd be tempted to rename this one paymentAndRefundHistoryHeading and have the old one as paymentHistoryHeading

Comment (singapaul) [reply it/testConstants/messages/PaymentHistoryMessages.scala:22]: done

File: it/controllers/PaymentHistoryControllerISpec.scala [hunk 3]
```diff
@@ -114,12 +116,48 @@ class PaymentHistoryControllerISpec extends ComponentSpecBase {
...
+
+        val result: WSResponse = IncomeTaxViewChangeFrontend.getPaymentHistory
+
+        Then("The Payment History page is returned to the user")
+        result should have(
+          httpStatus(OK),
+          pageTitleIndividual("paymentHistory.heading"),
+          elementTextBySelector("h1")(PaymentHistoryMessages.paymentHistoryHeadingFSOff)
+        )
+
```
[the remark this answers is not in the document]
Comment (singapaul) [reply it/controllers/PaymentHistoryControllerISpec.scala:148]: done

**Comment by singapaul** [comment]
Screenshot 2022-07-20 at 12 45 08

AT passed

**Pull request merged**